angular-workspace/lighthouserc.js

```diff
@@ -16,9 +16,7 @@ module.exports = {
         assert: {
             assertions: {
                 'categories:performance': ['warn', { minScore: 0.9 }],
-                // TODO: reset to 0.9 as part of https://github.com/ni/nimble/issues/1090
-                // TODO: reset to error as part of https://github.com/ni/nimble/issues/1650
-                'categories:accessibility': ['warn', { minScore: 0.88 }]
+                'categories:accessibility': ['error', { minScore: 0.8 }]
```

Comment:
> Removed the exisiting comment for an issue to get us back to 0.9 score as there are multiple issues impacting our score now.

Do we want to bring this number back up eventually? If so, any ideas on how we should remember to do that?

Reply:
I guess that is covered in https://github.com/ni/nimble/issues/280 which states a more ambitious 1.0 goal